klevia_sale/models/sale_order.py

```diff
@@ -15,3 +15,10 @@ def action_pdf_preview(self):
             'url': "/report/pdf/sale.report_saleorder/%s" % self.id,
             'target': 'new'
         }
+
+    @api.depends('sale_order_template_id')
+    def _compute_payment_term_id(self):
+        super()._compute_payment_term_id()
+        for order in self.filtered('sale_order_template_id'):
+            if order.sale_order_template_id.payment_term_id:
+                order.payment_term_id = order.sale_order_template_id.payment_term_id
```

Comment:
```suggestion
    @api.depends('sale_order_template_id')
    def _compute_payment_term_id(self):
        res = super()._compute_payment_term_id()
        for order in self.filtered('sale_order_template_id'):
            if order.sale_order_template_id.payment_term_id:
                order.payment_term_id = order.sale_order_template_id.payment_term_id
        return res
```


You need to return the value of super, even in compute methods